components/http_foundation.rst

```diff
@@ -585,6 +585,65 @@ represented by a PHP callable instead of a string::
 
 .. _component-http-foundation-serving-files:
 
+Streaming a JSON Response
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 6.2
+
+The :class:`Symfony\\Component\\HttpFoundation\\StreamedJsonResponse` class was
+introduced in Symfony 6.2. It allows an API to return a lot of data as JSON and keep
+the used resources low by make usage of Generators.
```

Comment:
We don't keep additional information in the version added directive itself, because version added directives will be removed in the next major version, so effectivly, the sentence: 

> It allows an API to return a lot of data as JSON and keep
 the used resources low by make usage of Generators.

will be gone

So I propose the following:
```suggestion
    The :class:`Symfony\\Component\\HttpFoundation\\StreamedJsonResponse` class was
    introduced in Symfony 6.2.
    
The :class:`Symfony\\Component\\HttpFoundation\\StreamedJsonResponse` class allows an API to return a lot of data as JSON and keep
the used resources low by make usage of Generators.
```